src/hooks/useListModules.ts

```diff
@@ -21,6 +30,32 @@ const useListModules = () => {
     [dispatch, setRequestId]
   );
 
+  const listDeliverables = useCallback(() => {
```

Comment:
I think we can make this into a `useEffect` and not export it, or at least add a `useEffect` block to call this so that we don't have to call the `useEffect` to update the list of modules in the component